ReleaseProcess.md

```diff
@@ -0,0 +1,9 @@
+**Process for releasing a new version of VSCode extension - Deploy to Azure**
+
+1. Update the new version in package.json and package-lock.json 
+2. Add one liner for the new version in ChangeLog.md and make changes in README.(if required).
```

Comment:
This need not be one liner. You can be more lines but should concisely tell what changed at a very high level